src/app/pages/network/forms/interfaces-form.component.ts

```diff
@@ -563,6 +564,15 @@ export class InterfacesFormComponent extends ViewControllerComponent implements
         this.entityForm.setDisabled('xmit_hash_policy', true, true);
       }
     });
+
+    this.entityForm.formGroup.get('lag_ports').valueChanges.pipe(untilDestroyed(this)).subscribe((value: string[]) => {
+      this.lag_ports.options.sort((a: Option, b: Option) => {
```

Comment:
We shouldn't re-order options upon every selection/un-select. It can be confusing to the user.